PhysicsTools/SelectorUtils/interface/VersionedIdProducer.h

```diff
@@ -102,16 +105,21 @@ VersionedIdProducer(const edm::ParameterSet& iConfig) {
     edm::LogWarning("IdInformation")
       << idmsg.str();
 
+    
+
     produces<std::string>(idname);
     produces<edm::ValueMap<bool> >(idname);
     produces<edm::ValueMap<float> >(idname); // for PAT
-    produces<edm::ValueMap<unsigned> >(idname);
+    produces<edm::ValueMap<unsigned> >(idname);  
+    produces<edm::ValueMap<unsigned> >(idname+std::string(bitmap_label));
```

Comment:
Hi Guys, no one would ever use TTree::Draw() with an edm::ValueMap, since you cannot use edm::Ptr/Ref within TTree::Draw() to begin with (and the syntax would break the Draw syntax afaik).